Eplant/views/eFP/efpFactory.tsx

```diff
@@ -0,0 +1,21 @@
+import EFP from ".";
+import { EFPViewerData } from "./Viewer/types";
+import CellEFP from "../CellEFP/cellEFP";
+
+
+export class efpFactory{
```

Comment:
This kind of pattern isn't really necessary in typescript, since modules can contain root-level functions. I think the code would look better if it `makeEfps` and `makeCellEfps` were not contained in a class